src/Missings.jl

```diff
@@ -207,4 +208,74 @@ missing
 """
 passmissing(f) = PassMissing{Core.Typeof(f)}(f)
 
-end # module
+
+"""
+   multiskipmissing(args...)
+
+
+Returns a tuple of iterators iterating over indices of elements `args`
+where all elements of `args` are not missing.
+
+# Examples
+```
+julia> x = [1, 2, missing, 4]; y = [1, 2, 3, missing];
+julia> tx, ty = multiskipmissing(x, y)
```

Comment:
Need to copy the output, which is interesting for users BTW. Also add a line break before (basically, just copy/paste the real output).